modules/planning/proto/planning_config.proto

```diff
@@ -354,4 +354,11 @@ message PlanningConfig {
   // default task config, it will be used if there is no scenario-specific
   // task config.
   repeated TaskConfig default_task_config = 5;
+  
+  optional string planning_pad_topic = 6;
```

Comment:
Just a trivial suggestion here. 
can you move the topic conf ahead of everything inside PlanningConfig?
otherwise they get buried by the big chunk of TaskConfig. just for readability and easy maintenance.

Reply:
Ok.